### ISSUE TITLE
Varying error message

### ISSUE THREAD

**Issue by StuartWheater** [datashield/DSI#2]
I have a test which checks what happens it 'dsAggregate' is called with out a method being specified.
That works well, I get a error message. But my slight problem is that the message changes. In CI on Azure Pipeline I get one message and on my machine when running devtools::test I get a different message:
"unable to find an inherited method for function 'dsAggregate' for signature '"NULL"'"
or
"unable to find an inherited method for function ‘dsAggregate’ for signature ‘"NULL"’"

One uses ‘ and ’, the other just uses '. Is there an encoding option I can set to get the
same behavior in both environments?

Stuart

**Comment by ymarcon** [comment]
It is a R error message, so I guess this is a R setting. Anyway, maybe Opal should be the one to raise the error. What is your dsAggregate call exactly?

**Comment by StuartWheater** [comment]
The test is just check the behavior of 'datashield.aggregate' if 'expr' is NULL, so the call is:

`datashield.aggregate(conns=ds.test_env$connections, expr=NULL)`

**Comment by ymarcon** [comment]
If it makes no sense to have such parameter, then `datashield.aggregate` should raise the error directly.

**Comment by StuartWheater** [comment]
Yes, the purpose of the test was just to check what the error message produced was. I have also now changed the test to use a regex not a fixed string.
